### DISCUSSION TITLE
Trigger Information for logging

### DISCUSSION THREAD

**Discussion by somasundaram1996** [operator-framework/java-operator-sdk#1677] [Q&A] [answered]
Hi I am trying to implement my custom logger for renconciliation. Is there a way to differentiate the Periodic reconciliation trigger and event based triggers?

**Comment by csviri** [comment, marked as answer]
Hi @somasundaram1996 , there is no, pls take a look here:
https://javaoperatorsdk.io/docs/faq#q-how-can-i-access-the-events-which-triggered-the-reconciliation
